src/main/cpp/util/path_windows.cc

```diff
@@ -378,10 +398,10 @@ std::string NormalizeWindowsPath(std::string path) {
     path = path.substr(4);
   }
 
-  static const std::string dot(".");
-  static const std::string dotdot("..");
+  static const std::basic_string<char_type> dot(1, '.');
+  static const std::basic_string<char_type> dotdot(2, '.');
```

Comment:
Ha, no need to use '.' and L'.' -- clever!

Reply:
Yes! Another workaround to make the code compile for both char and wchar_t.